### ISSUE TITLE
Update prod deploy for Apple IAP for VPN

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#13129]
﹍While﹍ [+https://api.accounts.firefox.com/v1/oauth/subscriptions/iap/plans/guardian-vpn+](https://api.accounts.firefox.com/v1/oauth/subscriptions/iap/plans/guardian-vpn) has Apple IAP config already (and the corresponding stage URL: [+https://api-accounts.stage.mozaws.net/v1/oauth/subscriptions/iap/plans/guardian-vpn+](https://api-accounts.stage.mozaws.net/v1/oauth/subscriptions/iap/plans/guardian-vpn)), there are a few tasks that need to happen to enable Apple IAP in production (assuming QA verifies everything in stage):

1. Enable Apple IAP via the feature flag in production
1. Update the Production endpoint in App Store Connect, so we receive App Store Server notifications
1. Add `bundleId` to the `guardian-vpn` document for IAP config ([https://github.com/mozilla-services/cloudops-infra/pull/4258](https://github.com/mozilla-services/cloudops-infra/pull/4258|smart-link) ) in prod Firestore, so we can use the correct App Store Server API credentials when making API calls.

The order here is important. We should not do #2 until #1 is done and deployed. #3, however, can be done anytime.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-5236)

**Comment by data-sync-user** [comment]
➤ Jon Buckley commented:

Github ↔︎ Jira sync gone awry

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

Bianca Danforth Jeremy Swinarton  Will this be a config flip (specifically, will this require an FxA deploy) to complete after integration is ready and tested? cc: Betty Fleming

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Yes for part 1. Part 2 is a config change in App Store Connect (an Apple website for managing iOS apps) which does not require a deploy.

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

Thank you! Jeremy Swinarton , as Part 1 of this ticket (Enable Apple IAP via the feature flag in production) will require a Firefox Accounts deploy, please keep us (myself, Bianca Danforth ) informed of your integration timelines and readiness. For a reference to our upcoming train schedule:

  * 239: Planning Aug 3 (today); Eng complete August 17; targets a prod deploy August 24.
  * 240: Planning August 17;  Eng complete August 31; targets a prod deploy Sept 7.

Can you inform us where you currently expect your integration to complete so that we can pull the ticket in to the appropriate sprint?

cc: Betty Fleming Rebecca Billings as this timeline will impact QA and launch strategies.

**Comment by data-sync-user** [comment]
➤ Jeremy Swinarton commented:

Hi Chelsea Lewis , thanks for the context. On our side, we are on target to be ready for QA by the end of the week. If all goes well we would like to deploy to production as soon as possible. It sounds like that will need to be after August 24, which is fine, but if you could pull this ticket into the current sprint that would be helpful. My apologies for not seeing this message last week!

**Comment by data-sync-user** [comment]
➤ Bianca Danforth commented:

Just coming full circle here; there should be no delays on the SubPlat side for QA testing to begin at the end of this week. I discovered an issue yesterday ( https://mozilla-hub.atlassian.net/browse/FXA-5034?focusedCommentId=578360 ), but I fixed it just now (PR ( https://github.com/mozilla-services/cloudops-infra/pull/4258 )) with jbuck’s help. Sorry for the confusion Jeremy.
